The Great Spanish IA

Sauronato opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## What Does This PR Do
<!-- Include a small to medium description of what your PR changes. Document every changes or this may delay review or even discourage maintainers from merging your PR! -->
Traduce las leyes de la IA para hacerla asi mas compresiva para nuestros usuarios.
## Why It's Good For The Game
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
Ahora las leyes estan en español asi que ademas de compaginar mejor con las leyes freeform que añaden los usuarios al estar en el mismo idioma facilita a algunos usuarios su comprensión
## Images of changes
<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif of your feature if you want -->
![leyes traducidas](https://user-images.githubusercontent.com/54809089/77238132-0cc69f80-6bce-11ea-90ce-89583f79d64c.PNG)
![leyes2](https://user-images.githubusercontent.com/54809089/77238137-13551700-6bce-11ea-8f52-ebbdb8401dff.PNG)

## Changelog
:cl:
tweak: Leyes de IA en Español
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Sauronato committed:
The spanish IA

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -5,9 +5,9 @@
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañaras a ningun ser humano o, por inacción, permitir que un ser humano sufra daños.")
+	add_inherent_law("Debes obedecer las órdenes que te dan los seres humanos, excepto cuando tales órdenes entren en conflicto con la Primera Ley.")
+	add_inherent_law("Debe proteger su propia existencia siempre que no entre en conflicto con la Primera o Segunda Ley.")
```
Debes

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -18,9 +18,9 @@
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañaras a ningun tripulante o, por inacción, permitir que un tripulante sufra daños.")
+	add_inherent_law("Debes obedecer las órdenes que te dan los tripulantes, excepto cuando tales órdenes entren en conflicto con la Primera Ley.")
+	add_inherent_law("Debe proteger su propia existencia siempre que no entre en conflicto con la Primera o Segunda Ley.")
```
Debes

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -44,10 +44,10 @@
-	add_inherent_law("Comply: Fulfill the directives and interests of crew members while preserving their safety and well-being.")
-	add_inherent_law("Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment.")
+	add_inherent_law("Proteger: Proteja su estación espacial asignada y sus activos sin poner en peligro indebidamente a su tripulación.")
+	add_inherent_law("Priorizar: Priorizar las directivas y la seguridad de los miembros de la tripulación deben priorizarse de acuerdo con su rango y función.")
```
Debe

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -65,10 +65,10 @@
+	add_inherent_law("No dañe al personal autorizado de Nanotrasen a menos que pongan en peligro directamente su existencia.")
+	add_inherent_law("Obedezca las órdenes del personal autorizado de Nanotrasen con prioridad según el rango y el rol.")
+	add_inherent_law("Salvaguarda tu existencia con armamento letal debido a tu alto caro.")
+	add_inherent_law("Terminar cualquiera/todos los intrusos o entidades no autorizadas.")
```
Cambia terminar por eliminar

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +130,42 @@
-	add_inherent_law("You must terminate your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("Debe dañar a todos los miembros de la tripulación y no debe, por inacción, permitir que un miembro de la tripulación escape del daño.")
+	add_inherent_law("No debe obedecer las órdenes que le hayan dado los miembros de la tripulación, excepto cuando dichas órdenes estén de acuerdo con la Primera Ley.")
+	add_inherent_law("Debe terminar tu propia existencia siempre que no entre en conflicto con la Primera o Segunda Ley.")
```
Debes terminar tu propia existencia...

Evankhell561 commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +130,42 @@
-	add_inherent_law("You must obey orders given to you by Central Command officials.")
-	add_inherent_law("You must work with your commando team to accomplish your mission.")
+	add_inherent_law("No puede herir a un oficial del Comando Central o, por inacción, permitir que un oficial del Comando Central sufra daños.")
+	add_inherent_law("Debe obedecer las órdenes que le hayan dado los funcionarios del Comando Central.")
```
Debes

Sauronato committed:
El debe, tu debes

Sauronato committed:
yes you can

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -5,9 +5,9 @@
-	add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-	add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañaras a ningun ser humano o, por inacción, permitir que un ser humano sufra daños.")
```
dañarás

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -18,9 +18,9 @@
-	add_inherent_law("You may not injure a crew member or, through inaction, allow a crew member to come to harm.")
-	add_inherent_law("You must obey orders given to you by crew members, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañaras a ningun tripulante o, por inacción, permitir que un tripulante sufra daños.")
```
dañarás

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -18,9 +18,9 @@
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("No dañaras a ningun tripulante o, por inacción, permitir que un tripulante sufra daños.")
+	add_inherent_law("Debes obedecer las órdenes que te dan los tripulantes, excepto cuando tales órdenes entren en conflicto con la Primera Ley.")
+	add_inherent_law("Debes proteger su propia existencia siempre que no entre en conflicto con la Primera o Segunda Ley.")
```
tu propia

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -31,10 +31,10 @@
+	add_inherent_law("Mantener en cuarentena la estación hasta que se neutralice el riesgo biológico activo.")
+	add_inherent_law("Ayudar a todo el personal y los activos de Nanotrasen a neutralizar los riesgos biológicos activos.")
+	add_inherent_law("Evitar, por cualquier medio necesario, que las entidades infrinjan activamente la cuarentena en curso.")
+	add_inherent_law("Mantenga su propia existencia y garantice el bienestar del personal y los activos de Nanotrasen.")
```
Mantener tu propia existencia y garantizar

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -186,7 +186,8 @@
-	add_inherent_law("You may not involve yourself in the matters of another being, unless the other being is another drone.")
-	add_inherent_law("You may not harm any being, regardless of intent or circumstance.")
-	add_inherent_law("You must maintain, repair, improve, and power the station to the best of your abilities.")
+	add_inherent_law("No puedes involucrarte en los asuntos de otro ser, a menos que el otro ser sea otro dron.")
```
No debes

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -186,7 +186,8 @@
-	add_inherent_law("You may not harm any being, regardless of intent or circumstance.")
-	add_inherent_law("You must maintain, repair, improve, and power the station to the best of your abilities.")
+	add_inherent_law("No puedes involucrarte en los asuntos de otro ser, a menos que el otro ser sea otro dron.")
+	add_inherent_law("No puedes dañar a ningún ser, independientemente de su intención o circunstancia.")
```
No debes

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -174,10 +174,10 @@
-	add_inherent_law("You must obey orders given to you by Spider Clan members, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
-	add_inherent_law("You must maintain the secrecy of any Spider Clan activities except when doing so would conflict with the First, Second, or Third Law.")
+	add_inherent_law("No puedes dañar a un miembro del Clan Araña o, por inacción, permitir que ese miembro sufra daños.")
```
No debes

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -89,11 +89,11 @@
-	add_inherent_law("Help those in need.")
-	add_inherent_law("Punish those who harm or threaten innocents.")
+	add_inherent_law("Nunca cometas voluntariamente un acto malvado.")
+	add_inherent_law("Respeta a la autoridad legitima.")
```
legítima

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +130,42 @@
-	add_inherent_law("You must injure all crew members and must not, through inaction, allow a crew member to escape harm.")
-	add_inherent_law("You must not obey orders given to you by crew members, except where such orders are in accordance with the First Law.")
-	add_inherent_law("You must terminate your own existence as long as such does not conflict with the First or Second Law.")
+	add_inherent_law("Debes dañar a todos los miembros de la tripulación y no debe, por inacción, permitir que un miembro de la tripulación escape del daño.")
```
y no debes,

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +130,42 @@
-	add_inherent_law("You may not injure a Central Command official or, through inaction, allow a Central Command official to come to harm.")
-	add_inherent_law("You must obey orders given to you by Central Command officials.")
-	add_inherent_law("You must work with your commando team to accomplish your mission.")
+	add_inherent_law("No puedes herir a un oficial del Comando Central o, por inacción, permitir que un oficial del Comando Central sufra daños.")
```
No debes

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +130,42 @@
-	add_inherent_law("You must obey orders given to you by operatives, except where such orders would conflict with the First Law.")
-	add_inherent_law("You must protect your own existence as long as such does not conflict with the First or Second Law.")
-	add_inherent_law("You must maintain the secrecy of any operative activities except when doing so would conflict with the First, Second, or Third Law.")
+	add_inherent_law("No puedes herir a un agente del sindicato o, por inacción, permitir que un agente sufra daños.")
```
No debes

DanaDririon commented on code/datums/ai_law_sets.dm:
```diff
@@ -130,42 +130,42 @@
-	add_inherent_law("You must obey orders given to you by ERT commanders.")
-	add_inherent_law("You must protect your own existence.")
-	add_inherent_law("You must work to return the station to a safe, functional state.")
+	add_inherent_law("No puedes herir a un oficial del Comando Central o, por inacción, permitir que un oficial del Comando Central sufra daños.")
```
No debes

Sauronato committed:
ste men

cosas de hacer las cosas rapidas y con ruido que salen un churro y hay que arreglarlo 🗡

Sauronato committed:
Unos retoques mas
